### PR TITLE
Change integrations rota

### PR DESCRIPTION
**Pull request by nixondante**
Updating the Integrations rota new Modular Connectors rota.

- Direct to #int-conjure-squad channel
- Link new GFH doc

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -9,24 +9,24 @@
     "message": "Hello! :wave: The Green Flag holder today is:\n`CDC:` <<2>>\n`Batch:` <<3>>\n\nUpdate rota <https://docs.google.com/spreadsheets/d/1XHDVi_VrUW0MX9emmpdqTsa78Xc6Qqpal24xidBXRb8/edit#gid=0|here>.\n\nSee details about what the Green Flag means <https://docs.google.com/document/d/1GOGUruD6ijSHyXgZLyJSRMyuSf0NVY1ICzl9PxaU208/edit|here>."
 },
 {
-    "name": "Integrations",
-    "description": "The integrations rota, posts to #int-incidents-flex-squad",
+    "name": "Modular Connectors",
+    "description": "The Modular Connectors rota, posts to #int-conjure-squad",
```
Comment (johnathanlaw) [inline config.json:13]: I don't think I can see this channel, is it a private one @nixondante ?

**Review by johnathanlaw — COMMENTED** [review]
Just checking on that one channel :) Otherwise approved.

File: config.json (hunk 1 above)
Comment (nixondante) [reply config.json:13]: Ahh yeah it's a private one:
image

Comment (nixondante) [reply config.json:13]: Just realised you needed to be added to the channel - done that now 😄

Comment (johnathanlaw) [reply config.json:13]: Yeah I need to be in it unfortunately! - Thanks for adding, thats it muted now :) Will squash and merge!

**Pull request merged**